action.py

```diff
@@ -191,6 +191,10 @@ def _fatal_help(msg):
             inputs.append(artifact)
 
 bundle_only = os.getenv("GHA_SIGSTORE_PYTHON_BUNDLE_ONLY") == "true"
+if not bundle_only:
+    _fatal_help("`bundle-only` is deprecated, this action no longer emits"
+                " other outputs by default")
```

Comment:
Actually, thinking about this more: let's just remove this option entirely, since we're doing a major bump.

(Sorry for the whiplash!)